docs/getting_started/first_steps.md

```diff
@@ -1,11 +1,184 @@
+```
+
+Then the Argo CD service will get its own separate IP address and optionally a DNS name if you have this option setup.
+
+If you use this method in a permanent manner, beware of the cost impact. A load balancer is usually priced separately on most cloud providers. Make sure you understand the cost implications of this method. An ingress is almost always a cost-effective method if you have multiple services in a single cluster.
+
+
+
+!!! warning
+    While this method is very easy to execute, we suggest you only use it for local clusters, product demos and proof-of-concept scenarios. It also opens your Argo CD installation to the whole world. Make sure that you  have a firewall and/or extra security constraints and know how users will access Argo CD.
+
+### Using Port forwarding
+
+Kubectl port-forwarding can also be used to connect to the API server without exposing the service.
+
+```bash
+kubectl port-forward svc/argocd-server -n argocd 8080:443
+```
+
+The API server can then be accessed using `https://localhost:8080`
+
+This method is expected to be temporary while you debug something or experiment locally. It is **NOT** recommended for production Argo CD installations.
+
+## Find the initial password
+
+On a default installation, Argo CD has one user defined with the following credentials
+
+* username = `admin`
+* password = randomly generated
+
+To find the password execute the following:
+
+```bash
+kubectl -n argocd get secret argocd-initial-admin-secret -o jsonpath="{.data.password}" | base64 -d; echo
+```
+
+!!! warning
+    You should delete the `argocd-initial-admin-secret` from the Argo CD
+    namespace once you changed the password. The secret serves no other
+    purpose than to store the initially generated password in clear and can
+    safely be deleted at any time. It will be re-created on demand by Argo CD
+    if a new admin password must be re-generated.
+
+## Use the CLI to deploy an application
+
+To start deploying resources you need to authenticate with the CLI and then create [Argo CD applications](../basics/apps/index.md).
+
+### CLI Login
+
+Using the username `admin` and the password from above, login to Argo CD's IP or hostname:
+
+```bash
+argocd login <ARGOCD_SERVER>
+```
+
+The value of `ARGOCD_SERVER` depends on the method you used in the previous section. It can be an IP address, a hostname such as `argocd.example.com` or even `localhost` if you use a local cluster.
+
+!!! note
+    The CLI environment must be able to communicate with the Argo CD API server. If it isn't directly accessible as described in the previous section, you can tell the CLI to access it using port forwarding through one of these mechanisms: 1) add `--port-forward-namespace argocd` flag to every CLI command; or 2) set `ARGOCD_OPTS` environment variable: `export ARGOCD_OPTS='--port-forward-namespace argocd'`.
+
+You can change the admin password using the command:
+
+```bash
+argocd account update-password
+```
+
```

Comment:
Maybe add an additional paragraph to state something like "In order to change the password of the admin user, you will need to know the current password. If you don't know the current password, you will have to use a different method to reset it."